ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -173,6 +174,33 @@ platform:
     username: "${GOVC_USERNAME}"
     ${RESOURCE_POOL_DEF}
 EOF
+else
+  cat >>"${CONFIG}" <<EOF
+baseDomain: $base_domain
+$MACHINE_POOL_OVERRIDES
+platform:
+  vsphere:
+    vcenters:
+    - datacenters:
+       - ${vsphere_datacenter}
+      password: ${GOVC_PASSWORD}
+      port: 443
+      server: ${vsphere_url}
+      user: ${GOVC_USERNAME}
+    failureDomains:
```

Comment:
This is not only for zone testing?

Reply:
This is not for zone testing, zone testing use step `ipi-conf-vsphere-zones` see [here](https://github.com/openshift/release/blob/e3ffe4ab4ceeaa3c851459087af6622f8e9b170c/ci-operator/step-registry/ipi/conf/vsphere/zones/ipi-conf-vsphere-zones-chain.yaml#L10C1-L11C1). 

After 4.15, the install config generated by default changed, following field are deprecated.

```
baseDomain: $base_domain
$MACHINE_POOL_OVERRIDES
platform:
  vsphere:
    vcenter: "${vsphere_url}"
    datacenter: "${vsphere_datacenter}"
    defaultDatastore: "${vsphere_datastore}"
    cluster: "${vsphere_cluster}"
    network: "${vsphere_portgroup}"
    password: "${GOVC_PASSWORD}"
    username: "${GOVC_USERNAME}"
    ${RESOURCE_POOL_DEF}
```